packages/lib-sourcify/src/Verification/VerificationTypes.ts

```diff
@@ -40,3 +56,67 @@ export type VerificationStatus =
   | 'extra-file-input-bug'
   | 'error'
   | null;
+
+export interface VerificationExport {
+  address: string;
+  chainId: number;
+  status: {
+    runtimeMatch: VerificationStatus;
+    creationMatch: VerificationStatus;
+  };
+  onchainRuntimeBytecode?: string;
+  onchainCreationBytecode?: string;
+  transformations: {
+    runtime: {
+      list: Transformation[];
+      values: TransformationValues;
+    };
+    creation: {
+      list: Transformation[];
+      values: TransformationValues;
+    };
+  };
+  deploymentInfo: {
+    blockNumber?: number;
+    txIndex?: number;
+    deployer?: string;
+    txHash?: string;
+  };
+  libraryMap: {
+    runtime?: StringMap;
+    creation?: StringMap;
+  };
+  compilation: {
+    language: CompilationLanguage;
+    compilationTarget: CompilationTarget;
+    compilerOutput: {
+      // The export should not include the AST object to reduce the size
+      sources?: Record<string, { id: number }>;
+    };
```

Comment:
Why here we are exporting only sources? I understand we want to reduce the size, but the full objects contain much more information:

```
export interface SolidityOutput {
  errors?: SolidityOutputError[];
  sources?: SolidityOutputSources;
  contracts: {
    [globalName: string]: {
      [contractName: string]: SolidityOutputContract;
    };
  };
}

export interface VyperOutput {
  compiler: string;
  errors?: VyperError[];
  sources: {
    [sourcePath: string]: VyperOutputSource;
  };
  contracts: VyperOutputContracts;
}
```

Reply:
Right